pkg/domain/entities/play.go

```diff
@@ -40,6 +40,9 @@ type PlayKubePod struct {
 	Containers []string
 	// Logs - non-fatal errors and log messages while processing.
 	Logs []string
+	// ContainerErrors - any errors that occurred while starting containers
+	// in the pod.
+	ContainerErrors []error
```

Comment:
I'm deliberately keeping this separate from Logs, because Logs being set doesn't change exit code, and this does

Reply:
This will not work via remote for the same reason as https://github.com/containers/podman/issues/8865

Reply:
Fixed